src/main/java/com/google/cloud/tools/maven/CloudSdkMojo.java

```diff
@@ -92,4 +95,8 @@ public String getCompileTargetVersion() {
     }
     return javaVersion;
   }
+
+  protected Plugin getPlugin() {
```

Comment:
Actually we can get it from the `pluginDescriptor` field. In any case I've removed it as it's not used anywhere, probably leftover from some fiddling around.